client/blocks/plan-thank-you-card/docs/example.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import sitesList from 'lib/sites-list';
+import PlanThankYouCard from 'blocks/plan-thank-you-card';
+
+const sites = sitesList();
+
+const PlanThankYouCardExample = React.createClass( {
```

Comment:
Nit: we could make this a stateless component and set `displayName` as a prop of it. See `blocks/post-item/docs/example` as an example.